perfkitbenchmarker/linux_benchmarks/object_storage_service_benchmark.py

```diff
@@ -1049,6 +1054,8 @@ def Prepare(self, vm):
     make_bucket_command = '%s mb' % vm.gsutil_path
     if FLAGS.object_storage_gcs_multiregion:
       make_bucket_command += ' -l %s' % FLAGS.object_storage_gcs_multiregion
+    if FLAGS.object_storage_storage_class:
```

Comment:
Should this be: "if FLAGS.object_storage_storage_class is not None:"? The same as the way you do the check above in the Azure case...